cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f08ac68",
+   "metadata": {},
+   "source": [
+    "# Itteratively Find Improved Consensus Labels for Multiannotator Data using Cleanlab"
```

Comment:
What do you think about title:

Iterative Use of Cleanlab to Improve Consensus Labels and Classification Models from Data Labeled by Multiple Annotators

Reply:
and Classification Models makes this a bit confusing for me. Is this two separate tasks? Are they related?

Would just Iterative Use of Cleanlab to Improve Consensus Labels from Data Labeled by Multiple Annotators be an ok change?

Reply:
I think improving classification models is the more valuable task though?  They are related because the classification models are improved because they're trained on the improved consensus labels. But I don't think the improved consensus labels alone are the primary goal, they are just a means to an end (improved model).

So what do you think about the below new title?

Iterative Use of Cleanlab to Improve Classification Models (and Consensus Labels) from Data Labeled by Multiple Annotators

Reply:
Yep this makes a lot more sense to me! Also shoes priority for model improvement vs consensus label improvement in an intuitive way